### ISSUE TITLE
trapped at home screen

### ISSUE THREAD

**Issue by kentburns** [DynamoDS/DynamoWishlist#269]
# Issue Description
Please fill in the following information to help us reproduce the issue:

## Dynamo version
Dynamo: `2.19.3.6394`

## Operating system
OS: `Microsoft Windows NT 10.0.19045.0`

## What did you do?
Clicked on Help>Display Start Page

## What did you expect to see?
Start page

## What did you see instead?
The start page, with no way to return to my graph. only option to save close and reopen. 

## Stack Trace
```
(From the Dynamo crash window select 'Details' -> 'Copy' and paste here)
```

---
CLR: `4.0.30319.42000`

**Comment by QilongTang** [comment]
To me it looks like the behavior of this feature should be refined or removed. I will discuss with out UX team. Thank you for reporting!

**Comment by QilongTang** [comment]
This is internally tracked as https://jira.autodesk.com/browse/DYN-7256. UX team has provided design and dev team will action on it soon. Stay tuned.

**Comment by avidit** [comment]
Fixed in Dynamo 3.4 https://github.com/DynamoDS/Dynamo/pull/15541
